app/validators/claim/base_claim_sub_model_validator.rb

```diff
@@ -40,7 +40,11 @@ def validate_has_many_associations_step_fields(record)
   end
 
   def validate_presence_of_association(association_name, options = {})
-    validate_presence(association_name, 'blank') if options && options[:presence]
+    if %i[warrant_fee].include? association_name
+      validate_presence(association_name, :blank) if options && options[:presence]
+    elsif options && options[:presence]
+      validate_presence(association_name, 'blank')
+    end
```

Comment:
This could maybe be a little cleaner. Something like this might work instead:

```    
    return unless options && options[:presence]

    if %i[warrant_fee].include? association_name
      validate_presence(association_name, :blank)
    else
      validate_presence(association_name, 'blank')
    end
```

(I haven't tested it though)